uml/sequence_diagrams/ClockView.txt

```diff
@@ -0,0 +1,46 @@
+@startuml
+participant DeactivatedClock 
+participant ClockView 
+participant BlueTooth
+participant Battery
+participant DisplayError 
+participant DisplayViews 
+
+opt Activation
+  DeactivatedClock -> ClockView:  Activate

```

Comment:
I'm wondering if we need Activation / Deactivation of the clock.

If you are in "Activate Clock" state of my display.png state machine, & the clock hasn't been activated, then what is happening on the display ?

Reply:
I think I used sometimes the word "clock" improperly,  activation here means activation of the smartwatch

Reply:
Changed name in the last commit